deployment/build-and-deploy-images.ps1

```diff
@@ -5,25 +5,34 @@
 
 Param(
     [string]
-    $ResourceGroupName
+    $AppResourceGroupName,
+
+    [string]
+    [Parameter(mandatory=$true)]
```

Comment:
nit: use of mandatory parameter could be consistent in general, not for this PR. Example, $AppResourceGroupName is not mandatory but I am sure we have this in other scripts so separate task will comb it together. Separate issue created https://github.com/Azure-Samples/distributed-az-edge-framework/issues/41

Reply:
Thanks Suneet, indeed let's track with the issue. I will set this one to resolved.